src/Parser/Warlock/Affliction/Modules/Features/ReapBuffTracker.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import Wrapper from 'common/Wrapper';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import { UNSTABLE_AFFLICTION_DEBUFF_IDS } from '../../Constants';
+
+const UA_IDS_SET = new Set(UNSTABLE_AFFLICTION_DEBUFF_IDS);
+
+class ReapBuffTracker extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  totalTicks = 0;
+  buffedTicks = 0;
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (!UA_IDS_SET.has(spellId)) {
+      return;
+    }
+    this.totalTicks += 1;
+    if (this.combatants.selected.hasBuff(SPELLS.DEADWIND_HARVESTER.id, event.timestamp)) {
+      this.buffedTicks += 1;
+    }
+  }
+
+  get unbuffedTicks() {
+    return this.totalTicks - this.buffedTicks;
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: (this.unbuffedTicks / this.totalTicks) || 1,  // if no UAs were cast (totalTicks and unbuffedTicks = 0), it should return NaN and thus be 1 (100% unbuffed ticks)
+      isGreaterThan: {
+        // TODO
```

Comment:
Is this comment left in intentionally? Do you want to update the numbers before it's merged or is it a thing for the future? (same comment is also in Haunt & MG suggestionThresholds)

Reply:
I plan on doing another round of getting feedback from the person that's helping me, so if it's ok with you, I'll leave the comment in. I'll remove it in next PR.